bpf/process/types/basic.h

```diff
@@ -560,10 +560,13 @@ filter_char_buf(struct selector_arg_filter *filter, char *args)
 		asm volatile("%[j] &= 0xff;\n" ::[j] "+r"(j)
 			     :);
 		length = *(__u32 *)&value[j];
-		asm volatile("%[length] &= 0x3f;\n" ::[length] "+r"(length)
+		asm volatile("%[length] &= 0x3f;\n" ::[length] "+r"(
+				     length)
```

Comment:
I think this change can be omitted, since it's just white space.

Reply:
sure, I was trying to pass the clang format test before

Reply:
yeah, tried to do the same and clang-format complained :(